history-of-quakers/modernized/19-chapter-17.adoc

```diff
@@ -307,6 +320,8 @@ subsisting in three distinct and separate persons`'?`' Whitehead
 and those with him denying this as an unscriptural doctrine,
 Vincent framed this syllogism.
 
+[quote]
+____
```

Comment:
aha! this will be a good place (for only the _second_ time ever, I think) to use the `[.syllogism]` class.  It's super rare so I wouldn't expect you to know/remember it, but you can see how it works near the bottom of this section:

https://flp-styleguide.netlify.app/#styling

The one gotcha is that each "paragraph" or sentence of the syllogism will need to be combined to be in one line, starting with an asterisk.  it's ok if these lines go over 100 characters, the linter won't bark because they start with a `* `